### ISSUE TITLE
Is there a Firefox version of this plugin?

### ISSUE THREAD

**Issue by hongyi-zhao** [cantino/selectorgadget#38]
Is there a Firefox version of this plugin?

Regards,
Zhao

**Comment by cantino** [comment]
You should be able to use the bookmarklet.

**Comment by hongyi-zhao** [comment]
I cannot find it, as shown below:

![image](https://user-images.githubusercontent.com/11155854/223472765-6a060012-39f0-4a05-9f2a-df331be45926.png)

**Comment by cantino** [comment]
Can you drag it to your bookmark bar from the website? https://selectorgadget.com/

**Comment by hongyi-zhao** [comment]
Thank you for letting me know this nice tip. It really works this way: Click the JavaScript function as URL indicated by the circle numbered 1 and 2. 

![image](https://user-images.githubusercontent.com/11155854/226145846-3741c9b7-ad04-4a07-9d01-d92f08af184f.png)

So, I can simply `Bookmark Link` in Firefox to quickly use them later.

But I still have some further questions:

1. The date `(updated August 7, 2013)` displayed on the above website indicates that the website has not been updated for a long time. So can the above method use the latest version of this tool? If not, how can I compile and use the latest GitHub version of this tool in Firefox?  
2. Call the tool from bookmarks library is not the same convenient as put a shortcut icon of it on the menu bar, similar to the following:

![image](https://user-images.githubusercontent.com/11155854/226146314-42ec2d88-07b7-4cfe-b5a6-84710bee386d.png)

I'm not sure if this is possible for this tool when called from within Firefox via cloud CDN based JavaScript call as represented above.

**Comment by cantino** [comment]
You're correct, it hasn't been updated in a decade. The core of the Chrome extension is the same code, just hosted in a slightly newer extension. This project isn't being developed at the moment.

**Comment by cantino** [comment]
Actually, that might not be true. The Chrome extension source might be slightly newer. Is the bookmarklet working for you?

**Comment by hongyi-zhao** [comment]
> This project isn't being developed at the moment.

But the latest commit has happened very recently:

https://github.com/cantino/selectorgadget/commit/6aa7560074f23736ba91066bba9c2b20ec219aa0

> Is the bookmarklet working for you?

I've done some simple tests, and it seems to work, but I don't know if it always works in very complicated cases.

**Comment by cantino** [comment]
I think there may have been once fix in 2014 that isn't in the bookmarklet, but everything newer is just how it's hosted as a chrome extension.

**Comment by hongyi-zhao** [comment]
So, the problem becomes the following: How can I rebuild the bookmarklet code snippet based on the latest GitHub version of this project?

**Comment by cantino** [comment]
You could look at the code in `build/` and compare it to the code that your browser loads when you run the bookmarklet. I'm sorry, I'm not interested in working on this codebase any more. You're welcome to run with it if you'd like.

**Comment by hongyi-zhao** [comment]
> I'm sorry, I'm not interested in working on this codebase any more.

Thank you for your hints again. Anyway, would you like to tell me the reason you made the above decision?

**Comment by cantino** [comment]
It's just not where my interest lies at the moment. Thanks for asking 😄 I'm always open to collaborators.

**Comment by hongyi-zhao** [comment]
What's the mean of `Go go` used by you as shown below?

![image](https://user-images.githubusercontent.com/11155854/226494016-63b7810c-2b1b-4401-b9de-b92c104b97e1.png)

**Comment by cantino** [comment]
https://en.wikipedia.org/wiki/Inspector_Gadget

**Comment by hongyi-zhao** [comment]
![image](https://user-images.githubusercontent.com/11155854/226494790-243d85dd-839e-4719-802d-6a035c3567c5.png)
![image](https://user-images.githubusercontent.com/11155854/226494835-0c90fb54-3778-40b8-8121-8cc5885b4163.png)
